src/lib/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: "../../.eslintrc.js",
+    rules: {
+      "no-console": "off",
+    }
+};
```

Comment:
Maybe it's enough to disable no-console per line?
I kinda like this rule (as warning), as I sometimes use `console.log` to debug manually and then forget to remove it before I commit. ;)